src/Navigation.js

```diff
@@ -1,42 +1,91 @@
 /** @jsxImportSource @emotion/react */
 
 import { Link, useLocation } from 'react-router-dom';
+import { NavIcon } from './NavIcon';
 
 export function Navigation() {
 	const { pathname } = useLocation();
 
-	const nav = [
-		{ name: 'Editor', url: '/' },
-		{ name: 'Settings', url: '/settings' },
-		{ name: 'About', url: '/about' },
+	const mainNav = [
+		{
+			name: 'New artwork',
+			url: '/new',
+			icon: 'a',
+		},
 	];
 
+	const secondaryNav = [
+		{
+			name: 'Settings',
+			url: '/settings',
+			icon: 'b',
+		},
+		{
+			name: 'About',
+			url: '/about',
+			icon: 'c',
+		},
+	];
+
+	const linkStyle = {
+		textDecoration: 'none',
+		':hover': {
+			textDecoration: 'none',
+		},
+	};
+
+	const listStyle = {
+		listStyle: 'none',
+		display: 'block',
+		padding: '0 5rem 0 0',
+		margin: 0,
+	};
```

Comment:
As soon as you encounter things you want to reuse, I would make a new component. I would not separate styles from the component (as we used to do back in the days) and keep things that work together, together.
Only separate concerns, not technologies.

So yeah we should have a primitive for lists.